projects/angular/src/modal/side-panel.ts

```diff
@@ -158,15 +183,31 @@ export class ClrSidePanel implements OnInit, OnDestroy, OnChanges {
     }
   }
 
-  private displaySideBySide() {
-    this.hostElement.classList.add('clr-side-panel-pinned-' + this.size);
+  private updateModalState() {
+    if (!this.modal) {
+      return;
+    }
+    if (this.pinned) {
+      this.modal.stopClose = true;
+      this.sideBySideMargins();
```

Comment:
Would it be better to name the function 'updateHostSize' (or something similar ) as it's updating the size class of the host element.
```suggestion
      this.updateHostSize();
      ```

Reply:
Done -> updatePinnedClassses